drivers/ieee802154/ieee802154_nrf5.h

```diff
@@ -82,4 +82,6 @@ struct nrf5_802154_data {
 	ieee802154_event_cb_t event_handler;
 };
 
+void nrf5_get_eui64(uint8_t *mac);
```

Comment:
should this function really be public? I think you may need to extend driver api for this. calling nrf5_get_eui64 from somewhere not within the nrf5 driver is not allowed.

Reply:
If the CONFIG_IEEE802154_NRF5_EUI64_CUSTOM_SOURCE is set to y, method has to be implemented by user, so I changed it to be extern and not share API in the header file.